.travis.yml

```diff
@@ -31,13 +30,16 @@ jobs:
     python: 3.6
     install: skip
     script: skip
+    env:
+    - PYPI_USER=amn41
+    - secure: "<encrypted PYPI_PASSWORD=password>"
```

Comment:
it's something @tmbo will need to add because I obviously don't know his password :)
it should be encrypted with `travis encrypt PYPI_PASSWORD=<password>`

Reply:
can't we re-use the old one?

Reply:
yes should be possible to reuse the old one

Reply:
no, we cannot :(
the encrypted string contains not only the password itself but also the name of the variable so it needs to be regenerated.
here is the information: https://docs.travis-ci.com/user/encryption-keys#usage

Reply:
It seems like this is fixed right?

Reply:
No, it currently contains my password :)

Reply:
But I will resolve the conversation and add these details below.